docs/docs/overview/quickstart/get-started-with-kotlin.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 9
+sidebar_label: Get started with Kotlin
+---
+
+# Kotlin Quickstart
+
+Learn how to integrate Novu into your Kotlin project on the fly and send Notifications across different channels (SMS, Email, Chat, Push).
+
+In this Quickstart, you’ll learn how to:
+
+- [Install and Set up the Novu Kotlin SDK.](#install-and-set-up-the-novu-kotlin-sdk-in-your-project)
+- [Configure a Notification Channel Provider.](#set-up-a-channel-provider)
+- [Set up a Notification Workflow.](#create-a-notification-workflow)
+- Create Subscribers and update a Subscriber's information.
+- Send your first Notification to a subscriber.
+- Send Notifications via Topics.
+
+Let's get started! :muscle:
+
+## Requirements
+
+To be able to use this Quickstart, you'll need to have the following:
+
+- A Novu account, if you don't have one yet, [sign up](https://web.novu.co) for free
+- A working Kotlin project running version 1.6.0+ and Kotlin compiler 1.8.0+
+
+The completed code for this quick start will be available on GitHub soon. Check it out [here](https://github.com/novuhq)
```

Comment:
Did you create the quickstart repo? If you did, let me so that you can transfer it to my account and I can transfer it to the Novu org.

Reply:
Alright, looking forward to it.